.github/workflows/gradle.yml

```diff
@@ -0,0 +1,53 @@
+name: "Publish to Gradle Gallery"
+# During the build-info release, the build-info JARs upload to the Maven Central repository.
+# The upload is asynchronous and may take few hours.
+# Since the Gradle plugin depends on the build-info JARs, we should publish it to the Gradle Gallery only after we make
+# sure the upload to Maven Central completed successfully.
+# To solve this out, this workflow runs every 4 hours to check if the Gradle plugin should be published to the Gradle
+# Gallery and publish the Gradle plugin if needed.
+on:
+  schedule:
+    # Once per 4 hours
+    - cron: "0 */4 * * *"
+jobs:
+  Publish_if_needed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Get Maven Central Latest Version"
+        run: |
+          export LATEST_MVN_CENTRAL_VERSION=`curl -s https://search.maven.org/solrsearch/select\?q\=g:"org.jfrog.buildinfo"+AND+a:"build-info-extractor-gradle" | jq -r '.response.docs[0].latestVersio'`
```

Comment:
@yahavi there is a typo in curl commnd :)
`jq -r '.response.docs[0].latestVersio`

Thats why scheduled job failed https://github.com/jfrog/build-info/runs/3080541646?check_suite_focus=true